resolwe_bio/processes/alignment/star.py

```diff
@@ -150,23 +150,105 @@ class ChimericReadsOptions:
             """Chimeric reads options."""
 
             chimeric = BooleanField(
-                label="Detect chimeric and circular alignments [--chimOutType SeparateSAMold]",
+                label="Detect chimeric and circular alignments",
                 default=False,
-                description="To switch on detection of chimeric (fusion) alignments (in addition "
-                "to normal mapping), --chimSegmentMin should be set to a positive value. Each "
-                "chimeric alignment consists of two segments.Each segment is non-chimeric on "
+                description="Detect chimeric and circular alignments. If set to True, "
+                "certain STAR parameters are set to values recommended for maximum sensitivity "
```

Comment:
My bad. The parameters change in bbduk_star process, not star. Perhaps a more correct wording would be

> If set to True, chimeric detection is turned on with default STAR parameters pertinent to chimeric detection. Output is either written to a .bam or separate (default) .sam file.